butter_cms/client.py

```diff
@@ -34,9 +35,11 @@ def api_get(self, slug='', params=None, path_override=None):
                 timeout=10,
             )
         except requests.exceptions.Timeout:
-            raise Timeout()
+            raise
 
-        if response.status_code != 200: raise HttpError(response.status_code)
+        # Avoid using raise_for_status(), more info here: https://github.com/ButterCMS/buttercms-python/issues/5
```

Comment:
In my opinion, we should copy the full reasoning here, instead of providing only the URL (URL could stay there as complementary to the comment, tho) because we can't be 100% sure that we won't migrate from GitHub in an indefinite future or lose issues history in some other way. The closer the explanation is to the code, the better.

That's my subjective opinion, however, so let me know @martinalbert @ViolanteCodes if you disagree.

Reply:
@Toreno96 i think this makes sense, and it looks like the reasoning line is pretty short. @martinalbert can this read:

`We are not using response.raise_for_status(), as it only raises errors for definite error codes; we want to raise errors for any status other than 200.`